x-pack/plugins/siem/public/components/stat_items/index.tsx

```diff
@@ -49,11 +52,81 @@ export interface StatItems {
 }
 
 export interface StatItemsProps extends StatItems {
-  key: string;
-  areaChart?: AreaChartData[];
-  barChart?: BarChartData[];
+  areaChart?: ChartConfigsData[];
+  barChart?: ChartConfigsData[];
 }
 
+export const addValueToFields = (
```

Comment:
Moving out these three functions from `useKpiMatrixStatus` and export these three functions for unit test:
https://github.com/elastic/kibana/blob/405f2eefbd37d504f41b1af9939a9ed1b1df1f82/x-pack/plugins/siem/public/components/stat_items/index.test.tsx#L159-L184